projects/go-lib/src/lib/components/go-radio/go-radio-button.component.scss

```diff
@@ -0,0 +1,38 @@
+@import '../../../styles/variables';
+
+input[type='radio'] {
+  display: none;
+}
+
+.go-radio__custom-radio {
+  display: inline-block;
+  width: 13px;
+  height: 13px;
+  border-radius: 100%;
+  cursor: pointer;
+  margin: 3px 3px 0px 5px;
+  border: 1px solid lighten($base-light-secondary, 13.5%);
+  background-color: $base-light;
+}
+
+input[type='radio']:enabled:hover + .go-radio__custom-radio {
+  box-shadow: 0 0 2px 1.5px lighten($base-light-secondary, 13.5%);
```

Comment:
This seems like an aggressive box-shadow. Also, we're introducing yet another box shadow. Can we make use of one of the ones that already exist in our `variables` file?

Reply:
This is the design Kim signed off on, so yes we are introducing another size 😄. I experimented with the ones we have predefined and they don't match the DSM specs or come close to it. Also the predefined box-shadows we have don't play nicely with a circular object like a radio button.

Reply:
Okay. Let's get her to take a look at these in the preview for sign off. I'll send it her way.

Reply:
This looks good to me. The box shadow is looking a little excessive right now, but I have submitted issues to replace both the button box shadow and the form controls (which includes radios, checkboxes and switches) box shadow from this to what is documented in the DSM and the issues. I imagine that this can be approved without changing the box shadow since that's a CSS issue and not hardcoded to the radio?

Reply:
Thanks Kim! We can handle changing this in the other PR's for form controls and buttons.